Changelog.md

```diff
@@ -301,7 +301,7 @@
 1. Require cmake 3.10.2, support `CXX_STANDARD` 17
     * [BitBucket pull request 68](https://osrf-migration.github.io/ignition-gh-pages/#!/ignitionrobotics/ign-cmake/pull-requests/68)
 
-    * [Full list of changes](https://github.com/gazebosim/gz-cmake/compare/ignition-cmake2_2.0.0...ign-cmake1)
+    * [Full list of changes](https://github.com/gazebosim/gz-cmake/compare/ignition-cmake2_2.0.0...ignition-cmake1)
```

Comment:
Oh these branch names shouldn't be changed

Reply:
The ignition-cmake1 branch doesn't exist anymore though :o